src/gen-use-table.py

```diff
@@ -47,6 +47,8 @@
 data[0][0x034F] = defaults[0]
 data[0][0x2060] = defaults[0]
 data[0][0x20F0] = defaults[0]
+data[0][0x11C44] = 'Consonant'
+data[0][0x11C45] = 'Consonant'
```

Comment:
Shouldn't these be `Consonant_Placeholder`?

Reply:
Please add github URL in comments as well.
Also, best if you also file a bug against https://github.com/roozbehp/unicode-data as well.

Reply:
Absolutely. `Consonant_Placeholder` makes more sense. I'll update and add URL.